cognite/neat/rules/importers/_rdf/_imf2rules/_imf2rules.py

```diff
@@ -4,25 +4,26 @@
 from pathlib import Path
 from typing import Literal, overload
 
-from rdflib import DC, DCTERMS, OWL, RDF, RDFS, SKOS, Graph
+from rdflib import DC, DCTERMS, OWL, RDF, RDFS, SH, SKOS, Graph
 
 from cognite.neat.rules.importers._base import BaseImporter, Rules
 from cognite.neat.rules.issues import IssueList
 from cognite.neat.rules.models import InformationRules, RoleTypes
 from cognite.neat.rules.models.data_types import _XSD_TYPES
 
-from ._owl2classes import parse_owl_classes
-from ._owl2metadata import parse_owl_metadata
-from ._owl2properties import parse_owl_properties
+from ._imf2classes import parse_imf_to_classes
+from ._imf2metadata import parse_imf_metadata
+from ._imf2properties import parse_imf_to_properties
 
 
-class OWLImporter(BaseImporter):
-    """Convert OWL ontology to tables/ transformation rules / Excel file.
+class IMFImporter(BaseImporter):
+    """Convert SHACL shapes to tables/ transformation rules / Excel file.
 
         Args:
-            filepath: Path to OWL ontology
+            filepath: Path to RDF file containing the SHACL Shapes
```

Comment:
Should this take in filepath? It looks like you immediately start up a graph. So suggest that the init takes a Graph, or a NeatStore, and then have a factory method for this.